usermods/usermod_v2_four_line_display/usermod_v2_four_line_display.h

```diff
@@ -168,13 +168,15 @@ class FourLineDisplayUsermod : public Usermod {
     // network here
     void setup() {
       if (type == NONE) return;
-      byte i;
-      if (type == SSD1306_SPI || type == SSD1306_SPI64) {
-        PinManagerPinType pins[5] = { { ioPin[0], true }, { ioPin[1], true}, { ioPin[2], true }, { ioPin[3], true}, { ioPin[4], true }};
-        if (!pinManager.allocateMultiplePins(pins, 5, PinOwner::UM_FourLineDisplay)) { type=NONE; return; }
-      } else {
-        PinManagerPinType pins[2] = { { ioPin[0], true }, { ioPin[1], true} };
-        if (!pinManager.allocateMultiplePins(pins, 2, PinOwner::UM_FourLineDisplay)) { type=NONE; return; }
+      PinManagerPinType pins[5];
+      auto pinCount = (type == SSD1306_SPI || type == SSD1306_SPI64) ? 2 : 5;
```

Comment:
Wow, you trust the compiler less than human choices?  😉 
I do note that `auto` is used for type declaration in other areas of core code, such as `hue.cpp`.
However, I will change this here, just to avoid leaving an open question.

Reply:
Indeed I am. But that is me, trying to perfect everything ... Not always successful though.
To be honest I do not like auto, you never know what compiler decides it is.